cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -23,6 +23,18 @@ declare namespace Deno {
    *  | "x11" (Linux)     | Xlib `Window` | Xlib `Display*` |
    *  | "wayland" (Linux) | `wl_surface*` | `wl_display*`   |
    *
+   * ```ts
```

Comment:
Adding `@example` keyword

```suggestion
   * @example 
   * ```ts
```